examples/src/bin/wifi_embassy_ble.rs

```diff
@@ -64,7 +64,7 @@ async fn main(_spawner: Spawner) -> ! {
 
     let esp_wifi_ctrl = &*mk_static!(
         EspWifiController<'static>,
-        init(timg0.timer0, Rng::new(peripherals.RNG),).unwrap()
+        init(timg0.timer0, Rng::new()).unwrap()
```

Comment:
I know it's a little bit out of scope of the PR, but if RNG can be constructed freely, we probably shouldn't bother passing it into esp-wifi.